ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -0,0 +1,229 @@
+//
+//  SampleHandler.swift
+//  BroadcastExtension
+//
+//  Created by Piyush Tank on 7/1/18.
+//  Copyright © 2018 Twilio. All rights reserved.
+//
+
+import Accelerate
+import TwilioVideo
+import ReplayKit
+
+class SampleHandler: RPBroadcastSampleHandler, TVIRoomDelegate, TVIVideoCapturer {
+
+    public var isScreencast: Bool = true
+
+    // Video SDK components
+    public var room: TVIRoom?
+    weak var captureConsumer: TVIVideoCaptureConsumer?
+
+    static let kDesiredFrameRate = 30
+    static let kDownScaledMaxWidthOrHeight = 640
+
+    let audioDevice = ExampleCoreAudioDevice()
+
+    public var supportedFormats: [TVIVideoFormat] {
+        get {
+            /*
+             * Describe the supported format.
+             * For this example we cheat and assume that we will be capturing the entire screen.
+             */
+            let screenSize = UIScreen.main.bounds.size
+            let format = TVIVideoFormat()
+            format.pixelFormat = TVIPixelFormat.formatYUV420BiPlanarFullRange
+            format.frameRate = UInt(SampleHandler.kDesiredFrameRate)
+            format.dimensions = CMVideoDimensions(width: Int32(screenSize.width), height: Int32(screenSize.height))
+            return [format]
+        }
+    }
+
+    override func broadcastStarted(withSetupInfo setupInfo: [String : NSObject]?) {
+
+        TwilioVideo.audioDevice = ExampleCoreAudioDevice(audioCapturer: self)
+
+        // User has requested to start the broadcast. Setup info from the UI extension can be supplied but optional.
+        let accessToken = "TWILIO-ACCESS-TOKEN";
```

Comment:
Consider using the setup view controller to deal with the token fetching. Fail the broadcast setup if fetching fails.